ultrasonic/src/main/AndroidManifest.xml

```diff
@@ -51,6 +59,16 @@
             android:exported="false">
         </service>
 
+        <service
+            android:name=".service.AutoMediaBrowserService"
```

Comment:
I've added this new service, but left the AutoMediaPlayerService in its place. I'm not sure if that should be rewritten entirely or if we can copy and reuse some parts from it.

Reply:
AutoMediaPlayerService is now removed.